<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Meteo Station</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="lumi">
           <property name="text">
            <string>Lumière</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="rain">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Pluie</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Activation relai</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Désactivation relai</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Temperature (en °C)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdTemperature">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="altitude">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Altitude (en m)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdAltitude">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Pression (en hPa)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdPressure">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Humidité du sol (en %)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdSoilHumidity">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="quantityofwater">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Précipitations (en mm)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdWater">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
